tests/articlerepo.test.ts

```diff
@@ -0,0 +1,101 @@
+import MockDate from 'mockdate';
+import { compareTrendiness } from '../src/repos/ArticleRepo';
+
+describe("testing getTrendingArticles query", () => {
+  beforeAll(() => {
+    //Set present time to a fixed day to compute trendiness from
+    MockDate.set('11-10-2020');
+  });
+
+  test('test correct trendiness order is returned', () => {
+    const articles =
+      [
+        {
+          id: 'one',
+          title: 'Yi Hsin is still meeting with publications',
+          publicationID: '5fac75af7f68e1f1a4ae8b1b',
+          articleURL: 'www.cuappdev.com/yi',
+          imageURL: '',
+          date: new Date('11-05-2020'),
+          shoutouts: 10
+        },
+        {
+          id: 'two',
+          title: 'who will the next appdev team lead be',
+          publicationID: '5fac75af7f68e1f1a4ae8b1b',
+          articleURL: 'www.cuappdev.com/lead',
+          imageURL: '',
+          date: new Date('11-08-2020'),
+          shoutouts: 10
+        },
+        {
+          id: 'three',
+          title: 'why coffee is better than tea',
+          publicationID: '5fac75af7f68e1f1a4ae8b1b',
+          articleURL: 'www.cuappdev.com/coffee',
+          imageURL: '',
+          date: new Date('11-07-2020'),
+          shoutouts: 10
+        },
+        {
+          id: 'four',
+          title: 'orko 3410 ta office hours',
+          publicationID: '5fac75af7f68e1f1a4ae8b1b',
+          articleURL: 'www.cuappdev.com/orko',
+          imageURL: '',
+          date: new Date('11-02-2020'),
+          shoutouts: 20
+        }
+      ];
+
+    const expectedOrder = [
+      {
+        id: 'two',
+        title: 'who will the next appdev team lead be',
+        publicationID: '5fac75af7f68e1f1a4ae8b1b',
+        articleURL: 'www.cuappdev.com/lead',
+        imageURL: '',
+        date: new Date('11-08-2020'),
+        shoutouts: 10
+      },
+      {
+        id: 'three',
+        title: 'why coffee is better than tea',
+        publicationID: '5fac75af7f68e1f1a4ae8b1b',
+        articleURL: 'www.cuappdev.com/coffee',
+        imageURL: '',
+        date: new Date('11-07-2020'),
+        shoutouts: 10
+      },
+      {
+        id: 'four',
+        title: 'orko 3410 ta office hours',
+        publicationID: '5fac75af7f68e1f1a4ae8b1b',
+        articleURL: 'www.cuappdev.com/orko',
+        imageURL: '',
+        date: new Date('11-02-2020'),
+        shoutouts: 20
+      },
+      {
+        id: 'one',
+        title: 'Yi Hsin is still meeting with publications',
+        publicationID: '5fac75af7f68e1f1a4ae8b1b',
+        articleURL: 'www.cuappdev.com/yi',
+        imageURL: '',
+        date: new Date('11-05-2020'),
+        shoutouts: 10
```

Comment:
might be useful to define a helper function for generating these objects to slim up the testing code and make it easier for generating entities in future tests